pkgs/applications/kde/kturtle.nix

```
@@ -3,7 +3,7 @@
 mkDerivation {
   pname = "kturtle";
   meta = with lib; {
-    homepage = "https://kde.org/applications/en/utilities/org.kde.kturtle";
+    homepage = "https://invent.kde.org/education/kturtle";
```

I saw similar PR (#151714) changing the homepage to apps.kde.org so I am not sure which one is better.

```suggestion
    homepage = "https://apps.kde.org/kturtle/";
```

---

personal preference for a repo, but that's me as a developer